opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -215,12 +215,28 @@ message DistributionValue {
     // The number of values in each bucket of the histogram, as described in
     // bucket_bounds.
     int64 count = 1;
-
-    // TODO: Add support for exemplars.
   }
 
   // If the distribution does not have a histogram, then omit this field.
   // If there is a histogram, then the sum of the values in the Bucket counts
   // must equal the value in the count field of the distribution.
   repeated Bucket buckets = 6;
+
+  // Exemplars are example points that may be used to annotate aggregated
+  // Distribution values. They are metadata that gives information about a
+  // particular value added to a Distribution bucket.
+  message Exemplar {
+    // Value of the exemplar point. It determines which bucket the exemplar
+    // belongs to.
+    double value = 1;
+
+    // The observation (sampling) time of the above value.
+    google.protobuf.Timestamp timestamp = 2;
+
+    // Contextual information about the example value.
+    map<string, string> attachments = 3;
```

Comment:
I'm still not sure about the name 'attachments' but I can't think of a better name offhand. On the up-side, we didn't go with 'context' :)

'annotations' is possible but  I don't want to confuse things with tracing.

Reply:
I chose this name "attachments" to be consistent with [Stackdriver](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries#exemplar) and Streamz. I think it's good to start with unless we came up with a better alternative.

Reply:
Understood - I think being consistent w/Stackdriver and Monarch is compelling.